src/sentry/models/group.py

```diff
@@ -622,6 +624,10 @@ class Meta:
             models.Index(fields=("status", "substatus", "id")),  # TODO: Remove this
             models.Index(fields=("status", "substatus", "first_seen")),
             models.Index(fields=("project", "status", "priority", "last_seen", "id")),
+            models.Index(
+                fields=("project", "seer_fixability_score", "seer_autofix_last_triggered")
+            ),
+            models.Index(fields=("project", "seer_autofix_last_triggered")),
```

Comment:
It takes a while to add migrations to this table, so you'll need to add these in a separate migration that is set to `is_post_deployment = True`. Then you can run it as a post deployment migration and let it run for a few hours to create these.

Do you need both indexes here? Are you likely to need to filter on both `project, seer_autofix_last_triggered` and `project, seer_is_fixable`? We're starting to have quite a few indexes on this table, so I want to make sure they're necessary

Reply:
The intention is that we will allow users to filter and sort issues on the issue stream by the fixability score and the last-triggered-date. Does that require indexes on these fields? (cc @trillville because you were asking for indexes)

If we do need the indexes, and I need a post-deployment migration, is the process to:
- remove the `models.Index(...)` definitions here
- remove the `db_index = True` params, e.g. `seer_autofix_last_triggered = models.DateTimeField(null=True, db_index=True)`
- introduce both in a follow up PR?

Reply:
Yeah, remove all the index defs from this prs, and move them to another.

The `db_index=True` params can be skipped. You are probably not going to be sorting/filter on this field by itself, only together with `project`, so these aren't too useful.
For filtering/sorting, so you want to be able to filter and sort by both of these fields? Is `last_triggered` a useful thing to be able to filter on here for a user? I can see the value in the score, but isn't the date really just an internal thing?

If you want, we can just discuss the indexing in the separate pr you make so that we can get this part merged.

Reply:
> For filtering/sorting, so you want to be able to filter and sort by both of these fields?

I think that was the ask from @rachrwang and @trillville, so I'll let them chime in on the product requirements.

But yeah, I'll remove the indexing here and we can defer that discussion to a future PR.

Reply:
`last_triggered` would be something i can see us sorting and filtering on - for example we want to filter to only issues that have recent runs, or sort by the most recent run (like you are trying to find one that you were just looking at)

having said that, i wouldn't say its a dealbreaker or something, especially if we want to be careful about adding too many indexes to this table @wedamija